packages/perseus/src/widgets/interactive-graphs/types.ts

```diff
@@ -14,6 +14,32 @@ export type MafsGraphProps<T extends InteractiveGraphState> = {
     dispatch: (action: InteractiveGraphAction) => unknown;
 };
 
+export type InteractiveGraphName =
+    | "angle"
+    | "segment"
+    | "linear-system"
+    | "linear"
+    | "ray"
+    | "none"
+    | "polygon"
+    | "point"
+    | "circle"
+    | "quadratic"
+    | "sinusoid";
+
+export type InteractiveGraphEnum =
+    | "ANGLE"
+    | "SEGMENT"
+    | "LINEAR_SYSTEM"
+    | "LINEAR"
+    | "RAY"
+    | "NONE"
+    | "POLYGON"
+    | "POINT"
+    | "CIRCLE"
+    | "QUADRATIC"
+    | "SINUSOID";
+
```

Comment:
**suggestion**: I'm guessing the `InteracativeGraphEnum` matches the enum values for the event in CEDAR?  In that case, I'd recommend that this mapping live in webapp's adapter rather than here. Otherwise we're pulling CEDAR concepts into the Perseus codebase. I think it's completely fine to use the `interactive-graph`'s graph type in the event and then map it to the CEDAR value in webapp.